test/e2e/tests/editor.spec.js

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
Sorry. Keep finding little things...

Would it make sense to call these files the same as their associated views?

```
/       >>> browse
/edit   >>> edit
/sheet  >>> sheet
/media  >>> media
/config >>> config
```

Reply:
Not sure if it will scale this way forever. 😄